diff --git a/Services/Repository/PluginSlot/class.ilObjectPluginGUI.php b/Services/Repository/PluginSlot/class.ilObjectPluginGUI.php
index b395a6b0ea..d547103806 100644
--- a/Services/Repository/PluginSlot/class.ilObjectPluginGUI.php
+++ b/Services/Repository/PluginSlot/class.ilObjectPluginGUI.php
@@ -159,7 +159,7 @@ abstract class ilObjectPluginGUI extends ilObject2GUI
                 $this->ctrl->forwardCommand($gui);
                 break;
             default:
-                if ($cmd === "save" && $this->getCreationMode()) {
+                if ($cmd === "save" || $this->getCreationMode()) {
                     $this->$cmd();
                     return;
                 }
